TODO.org

```diff
@@ -26,6 +26,16 @@
 
 - [ ] automated testing with multiple emacs versions
 
+* Movement
+
+- [ ] rework keybindings to be consistent with more popular modes
+
+- [ ] document keybidning/movement in general
+
+- [ ] beginning/end of defun should handle top-level statement block?
+
+- [ ] why is imenu binded?
```

Comment:
If you mean the key-binding, I added that with the same key-binding that the modern python.el uses so that there was parity between the two modes. Is was only in Emacs 29 that Imenu got a default key-binding.